jetty-server/pom.xml

```diff
@@ -4,16 +4,20 @@
     <artifactId>jetty-project</artifactId>
     <version>10.0.0-SNAPSHOT</version>
   </parent>
+
   <modelVersion>4.0.0</modelVersion>
   <artifactId>jetty-server</artifactId>
   <name>Jetty :: Server Core</name>
   <description>The core jetty server artifact.</description>
   <url>http://www.eclipse.org/jetty</url>
+
   <properties>
     <bundle-symbolic-name>${project.groupId}.server</bundle-symbolic-name>
   </properties>
+
   <build>
     <plugins>
+      <!--TODO: do we need this test-jar?-->
```

Comment:
I don't think so. But maybe leave it temporary until we finish fixing #3080 . Well this will change the jars we publish to Maven central. But is there anyone using http://repo.maven.apache.org/maven2/org/eclipse/jetty/jetty-server/9.4.14.v20181114/jetty-server-9.4.14.v20181114-tests.jar ?